src/main/scala/net/ripe/rpki/publicationserver/AppConfig.scala

```diff
@@ -54,6 +54,14 @@ class AppConfig {
   lazy val publicationServerTrustStorePassword = getConfig.getString("publication.server.truststore.password")
   lazy val storePath = getConfig.getString("xodus.path")
 
+  // no delay by default
+  lazy val notificationWritingDelay = {
+    val field = "publication.server.notification.writing.delay"
+    if (getConfig.hasPath(field))
+      getConfig.getInt(field)
+    else 0
```

Comment:
Why not default to 10?

Reply:
Because it breaks tests :)

Reply:
I think it's reasonable behaviour -- no delay by default :)
Otherwise: why 10 and not something else?

Reply:
10 is the minimum value allowed for caching. I'd be fine with 10 or 15 as defaults